Exercise06/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise06
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var n = 100;
+
+            var numbers = Enumerable.Range(2, n);
+            var primeNumbers = numbers.TakeWhile(x => x < n)
+                .Where(x => !numbers.TakeWhile(y => y <= (int)Math.Sqrt(x))

```

Comment:
Although the result is correct, the solution is still bad. Try to make it better. Bellow is my solution
```C#
            var n = 100;

            var numbers = Enumerable.Range(2, n-1).ToList();

            var primeNumbers = numbers
                .Where(x => Enumerable.Range(2, (x - 1)/2).All(y => x % y > 0))
                .ToList();

            primeNumbers.ForEach(Console.WriteLine);

            Console.ReadKey();
```

Reply:
Done